tests/queries/0_stateless/02801_backup_native_copy.sh

```diff
@@ -8,64 +8,68 @@ CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 
 set -e
 
-$CLICKHOUSE_CLIENT -m -q "
+client_opts=(
```

Comment:
It cost me some time to recognize that the `client_opts` is used later globally.

If @nickitat is OK to merge, we can leave it for later. Although we can overwrite `CLICKHOUSE_CLIENT="$CLICKHOUSE_CLIENT --send_logs_level error"`.